api/v1alpha3/awsmachine_webhook_test.go

```diff
@@ -217,3 +218,49 @@ func TestAWSMachine_ValidateUpdate(t *testing.T) {
 		})
 	}
 }
+
+func TestAWSMachine_Default(t *testing.T) {
+	machine := &AWSMachine{
+		Spec: AWSMachineSpec{
+			ProviderID:               nil,
+			AdditionalTags:           nil,
+			AdditionalSecurityGroups: nil,
+		},
+	}
+
+	tests := []struct{
+		name string
+		cloudInit CloudInit
+		assertFunc func(*AWSMachine, *WithT)
```

Comment:
This works, but I find it preferable in simple comparisons like this to have something like `expected string` and have a `g.Expect` comparison directly in the subtest, instead of using a function pointer.